makedist

```diff
@@ -35,6 +37,9 @@ while true; do
     elif [ "$1" = "-m" ]; then
         MACHTYPE="$2"
         shift 2
+    elif [ "$1" = "-3" ]; then
+        FUSE3=true
+        shift
```

Comment:
I'm not very happy with an option to do this.  Can't this be discovered automatically somehow?  Maybe looking for fusermount3 like is done in umountrepo?

Reply:
The problem is that you may need to create a distribution on another machine to be used on HPC node.
In my case fuse2 was available on login node but not on worker nodes. So there should be some way to force this.
It can be done by setting environment variable if option is not desired.

Reply:
No, I don't prefer an environment variable, I just prefer to have a default that's right most of the time.  Let's have it default to fuse3 if fusermount3 is found on the host where makedist is run, and have `-3` and `-2` options to override the default.

Reply:
The new CI checks look good, but I'd still like to see the suggestion I had above:

> Let's have it default to fuse3 if fusermount3 is found on the host where makedist is run, and have `-3` and `-2` options to override the default (for the non `-s` case).